### ISSUE TITLE
interpret output files

### ISSUE THREAD

**Issue by FanBioinformatics** [njaupan/ECDNA#7]
Dear njaupan,

Can you briefly explain those files in 
folder containing peak calling files (peak_files)?

Any chance I can use information contained in this folder to trace back the read sequence and subset them out from the original fastq files?

Best,
Huihui

**Comment by njaupan** [comment]
Yes, You can trace back the potential reads from bed files in this folder:

the *site file ( eg. ecc.sr.site) shows all potential peaks
the *split.bed ( eg. ecc.sr.split.bed)show all potentially split reads mapped on the genome, with read names in 6th column.
the *disc.bed ( eg. ecc.sr.disc.bed) show all potentially discordant reads mapped on the genome, with read names in 6th column.

Best,
panpan

**Comment by FanBioinformatics** [comment]
Thanks Panpan.
